book/website/community-handbook/communication-channels.md

```diff
@@ -1,9 +1,76 @@
+This platform enables instant messaging, fostering quick exchanges and real-time collaborations among team members.
+It is a space where folks ask questions, learn about each other’s work, and get involved with _The Turing Way_ and other projects. 
+While it is not the only communication channel used by the community, it is a popular platform, increasingly used by remote teams around the world.
+
+You can join the workspace using the [joining invitation](https://theturingway.slack.com/join/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw#/shared-invite/email).
+
+### Social media channels
+
+In expanding our reach beyond project-specific communication channels, _The Turing Way_ project delivery team uses social media platforms to connect with a wider audience and share project updates broadly.
+
+- **Twitter (@turingway):** Used since 2020, our Twitter presence has played an important role in getting the community engaged in real-time with updates and announcements from the project. Follow [@turingway](https://twitter.com/turingway).
+- **Fosstodon (@turingway):** Since 2023, we have been using [Fosstodon](https://fosstodon.org/@turingway), a platform for open science practitioners in our community.
+- **LinkedIn Page:** Since late 2023, we have been testing and expanding our presence on [LinkedIn](https://www.linkedin.com/company/the-turing-way) among professional networks.
+
+These social media platforms complement our primary communication channels, providing additional avenues (often requiring manual cross-posting) for community engagement and outreach.
+
+## Asynchronous Communications
+
+In addition to our synchronous communication channels, we leverage various platforms for archiving, sharing, and preserving project-related outputs within _The Turing Way_ community, which serve as asynchronous communication channels.
+
+### Monthly Newsletters
+
+Asynchronous communication is facilitated through {ref}`monthly newsletters<ch-newsletters>`, offering updates on the project's progress, achievements, and upcoming events. These newsletters provide a digestible format for community members to stay informed at their own pace, offering a snapshot of recent developments within _The Turing Way_.
+
+### Zenodo Community Page
+
+_The Turing Way_ community page on [Zenodo](https://zenodo.org/communities/the-turing-way) is the main platform to archive and preserve project-related outputs, including slide decks, reports, datasets (such as the entire GitHub repository and illustrations), and publications. This platform ensures stable and citable records, enhancing accessibility and longevity.
+
+### GitHub Repositories
+
+Central to our information storage and sharing, our [GitHub repositories](https://github.com/the-turing-way) serve as the primary hub for project-related documents, resources, and collaborative efforts, offering transparency and collaboration for both synchronous and asynchronous communication.
+
```

Comment:
```suggestion

Since our transition from a single repository to a GitHub organisation, we are moving towards community-maintained repositories on various projects taking place within the community, including governance work, Working Groups, community events and different kinds of communication outputs.

```